com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Recursive-Rendering.md

```diff
@@ -41,3 +41,5 @@ Since recursive rendering uses an independent render pass, HDRP cannot render an
 | **Max Depth**  | Controls the maximum number of times a ray can reflect or refract before it stops and returns the final color. Increasing this value increases execution time exponentially. |
 | **Max Ray Length** | Controls the length of the rays that HDRP uses for ray tracing. If a ray doesn't find an intersection, then the ray returns the color of the sky. |
 | **Min Smoothness** | Defines the threshold at which reflection rays are not cast if the smoothness value of the target surface is inferior to the one defined by the parameter. |
+| **Ray Miss**                  | Determines what HDRP does when recursive rendering (RR) doesn't find an intersection. Choose from one of the following options: <br/>&#8226;**Reflection probes**: HDRP uses reflection probes in your scene to calculate the last RR bounce.<br/>&#8226;**Sky**: HDRP uses the sky defined by the current [Volume](Volumes.md) settings to calculate the last RR bounce.<br/>&#8226;**Both** : HDRP uses both reflection probes and the  the sky defined by the current [Volume](Volumes.md) settings to calculate the last RR bounce.<br/>&#8226;**Nothing**: HDRP does not calculate indirect lighting when RR doesn't find an intersection.<br/><br/>This property is set to **Both** by default|
+| **Last Bounce**               | Determines what HDRP does when recursive rendering (RR) lights the last bounce. Choose from one of the following options: <br/>&#8226;**Reflection probes**: HDRP uses reflection probes in your scene to calculate the last RR bounce.<br/>&#8226;**Sky**: HDRP uses the sky defined by the current [Volume](Volumes.md) settings to calculate the last RR bounce.<br/>&#8226;**Both**:  HDRP uses both reflection probes and the sky defined by the current [Volume](Volumes.md) settings to calculate the last RR bounce.<br/>&#8226;**Nothing**: HDRP does not calculate indirect lighting when it evaluates the last bounce.<br/><br/>This property is set to **Both** by default. |
```

Comment:
As above: 

| **Last Bounce**               | Determines what HDRP does when recursive rendering lights the last bounce. Choose from one of the following options: <br/>&#8226;**Reflection probes**: HDRP uses reflection probes in your scene to calculate the last bounce.<br/>&#8226;**Sky**: HDRP uses the sky defined by the current [Volume](Volumes.md) settings to calculate the last recursive rendering bounce.<br/>&#8226;**Both**:  HDRP uses both reflection probes and the sky defined by the current [Volume](Volumes.md) settings to calculate the last recursive rendering bounce.<br/>&#8226;**Nothing**: HDRP does not calculate indirect lighting when it evaluates the last bounce.<br/><br/>This property is set to **Both** by default. |